lib/datasource/bitbucket-tags/index.ts

```diff
@@ -111,11 +111,11 @@ export class BitBucketTagsDatasource extends Datasource {
       BitBucketTagsDatasource.getCacheKey(registryUrl, lookupName, 'digest'),
   })
   override async getDigest(
-    { lookupName: repo, registryUrl }: DigestConfig,
+    { lookupName: repo }: DigestConfig,
```

Comment:
Why dropping `registryUrl` from this?

Reply:
It was an error. It's unused parameter, but is used to form cache key

Reply:
@viceice How do you think, would it be hard to make decorators enforce the signature of decorated function

Reply:
Yes, we couldn't find a solution yet and given up on this yet. It will maybe possible with a future typescript version.